### ISSUE TITLE
he.decode is not a function inside production ssr mode

### ISSUE THREAD

**Issue by minht11** [natemoo-re/astro-remote#5]
When running built project I get error `he.encode is not a function`

Manually inserting console.log inside dist/server/entry.mjs file 'he' module functions are exported inside default namespace
```
[Module: null prototype] {
  default: {
    version: '1.2.0',
    encode: [Function: encode] { options: [Object] },
    decode: [Function: decode] { options: [Object] },
    escape: [Function: escape],
    unescape: [Function: decode] { options: [Object] }
  }
}
```
To fix this I tried adding 'he' to noExternal which fixes it during production, but breaks dev mode
```js
export default defineConfig({
  vite: {
    ssr: {
      noExternal: ['he'],
    },
  },
})

```

**Comment by natemoo-re** [comment]
Hmm thanks for the issue! I'll try to take a look at this soon.

**Comment by denniscampos** [comment]
I had the same issue with a slight different error: 
```
▶ src/pages/[slug].astro
 error   he.encode is not a function
  Please report this to https://github.com/markedjs/marked.
  File:
    C:\Users\...\...\node_modules\marked\lib\marked.esm.js:2688:41
  Code:
    2687 |         renderer[prop] = (...args) => {
    > 2688 |           let ret = pack.renderer[prop].apply(renderer, args);
           |  
```

I tried the above method although I'm not using SSR. I'm using via webhooks to trigger any `published` blogs. 

When the articles are all `unpublished` it builds fine. Just weird adding what the OP provided worked. 

Also aware it says to report to markedjs but I don't have that installed and when looking in my package-lock it shows via astro-remote

```
    "node_modules/astro-remote": {
      "version": "0.2.3",
      "resolved": "https://registry.npmjs.org/astro-remote/-/astro-remote-0.2.3.tgz",
      "integrity": "sha512-vsY736YjWhpFgx4KUxCBdK0QJmOk0W61VQwO7v6qmfGdIxZyx6N7hBNou57w2mw68hQSe5AbRs602pi05GDMHw==",
      "dependencies": {
        "he": "^1.2.0",
        "marked": "^4.0.18",
        "ultrahtml": "^0.1.1"
      }
    },
```

**Comment by minht11** [comment]
While this is fixed, I was able to make it work in dev/prod modes with the following config.
```ts
export default defineConfig({
vite: {
    ssr: {
      noExternal: ['he'],
      optimizeDeps: {
        include: ['astro-remote', 'he'],
        disabled: 'build',
      },
    },
  },
}
```

**Comment by natemoo-re** [comment]
`he` has been replaced by `entities` thanks to @JulianCataldo in https://github.com/natemoo-re/astro-remote/pull/9. Released as [`astro-remote@0.2.4`](https://github.com/natemoo-re/astro-remote/releases/tag/astro-remote%400.2.4)
